### DISCUSSION TITLE
zk-message

### DISCUSSION THREAD

**Discussion by baidang201** [zkp-co-learning/zkp-co-learn#61] [I've got an idea!]
一句话 IDEA：
 基于polygan zkevm 链上消息的from和to的隐藏，但from和to账户可以知道消息是自己的

技术栈：
（circom snarkjs solidity typejs）

我的角色：技术设计，项目/产品经理，项目部署
人数需求（不包含发起者）：共需3人，1人 web3前端经验  2人zk及solidity开发者
我的联系方式：tg smithwow
个人简介：5年区块链开发，有rust/nodejs经验，zp刚入门

现阶段3-4人的团队较为合适, 期待加入, 一起交流和学习zkp

参考：
https://vivianblog.hashnode.dev/how-to-create-a-zero-knowledge-dapp-from-zero-to-production

**Comment by SMARTJIANG2022** [comment]
<img width="1117" alt="a5399e52bd7df14c85ae96b44760751" src="https://user-images.githubusercontent.com/113282009/224553880-7c6c1c3a-615c-489d-a760-a84fb4c0c056.png">
您好， 我在步骤五中使用了Plonk 和 Groth16都报同一个错误，请问是否遇到过相同问题，怎么处理？谢谢

**Reply by baidang201** [reply]
没跑到这一步，再看
